Formula/vsftpd.rb

```diff
@@ -44,11 +44,45 @@ def caveats
       s += <<-EOS.undent
         vsftpd was compiled with SSL support. To use it you must generate a SSL
         certificate and set 'enable_ssl=YES' in your config file.
+
       EOS
     end
 
+    s += <<-EOS.undent
+      To use chroot, vsftpd requires root privileges, so you will need to run
+      `sudo vsftpd`.
+      You should be certain that you trust any software you grant root privileges.
+
+      The vsftpd.conf file must be owned by root in order to work correctly:
```

Comment:
Yes - vsftpd will refuse to start with an error, `500 OOPS: config file not owned by correct user, or not a file`. I'll state that directly in the caveat. Is there a supported/recommended way of writing a script to handle setup? When I looked through existing formula, the only similar examples I found used a caveat to advise the user to `chown`.